<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Risk Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>901</width>
      <height>701</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>601</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>This is a program that calculates the risk percentage of each stock in 3 difrent ways.</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button1">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>640</y>
        <width>91</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Beta Coef</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="stock_name">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>230</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>170</y>
        <width>591</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Write the name of the stock below so you can see the calculations graphicaly:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>200</y>
        <width>571</width>
        <height>421</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>transparent2.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="button2">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>640</y>
        <width>91</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Shape Ratio</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button3">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>640</y>
        <width>91</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Value at risk</string>
      </property>
     </widget>
     <widget class="QPushButton" name="load_button">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>230</y>
        <width>91</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Load Data</string>
      </property>
     </widget>
     <zorder>label_3</zorder>
     <zorder>label</zorder>
     <zorder>button1</zorder>
     <zorder>stock_name</zorder>
     <zorder>label_2</zorder>
     <zorder>button2</zorder>
     <zorder>button3</zorder>
     <zorder>load_button</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
